dotCMS/src/main/java/com/dotcms/security/apps/AppsUtil.java

```diff
@@ -0,0 +1,150 @@
+package com.dotcms.security.apps;
+
+import com.dotcms.rest.api.v1.DotObjectMapperProvider;
+import com.dotmarketing.exception.DotDataException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.ByteStreams;
+import com.liferay.util.Base64;
+import com.liferay.util.Encryptor;
+import com.liferay.util.EncryptorException;
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.ArrayList;
+import java.util.List;
+import javax.crypto.Cipher;
+import org.apache.commons.io.input.CharSequenceInputStream;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class AppsUtil {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)